espnet2/bin/enh_scoring.py

```diff
@@ -29,6 +29,7 @@ def scoring(
     ref_scp: List[str],
```

Comment:
It’s good to make a unit test and cover this function.

Reply:
Added the test